tests/spread/plugins/colcon/snaps/colcon-talker-listener/src/listener_py/listener.py

```diff
@@ -1,18 +1,24 @@
 import rclpy
 from rclpy.node import Node
+from rclpy.qos import QoSProfile
+from rcl_interfaces.msg import ParameterDescriptor
 from std_msgs.msg import String
 
 
 class Listener(Node):
     def __init__(self):
         super().__init__("listener")
 
-        self._subscription = self.create_subscription(String, "chatter", self._callback)
-
-        parameter = self.get_parameter("exit-after-receive")
+        self.declare_parameter("exit_after_receive", False, ParameterDescriptor())
```

Comment:
I believe this can be a one-liner  
```cpp
parameter = self.declare_parameter("exit_after_receive", False, ParameterDescriptor())
```

Reply:
Huh, right you are, I didn't realize `declare_parameter` also retrieved the value. Man this API is odd. Fixed!